bot/exts/evergreen/music/__init__.py

```diff
@@ -0,0 +1,13 @@
+import logging
+
+from discord.ext import commands
+
+from bot.exts.evergreen.music.app.cog import Music
+
+
+logger: logging.Logger = logging.getLogger(__name__)
```

Comment:
Logging is not required.

Reply:
You should look at the previous reviews and not repeat the same comments. Most of your comments are repeated here.

Reply:
As commented on Shivansh' review, this is now used